homeassistant/components/wallbox/const.py

```diff
@@ -43,6 +44,23 @@ class WallboxSensorEntityDescription(SensorEntityDescription):
     precision: int | None = None
 
 
+@dataclass
+class WallboxNumberEntityDescription(NumberEntityDescription):
+    """Describes Wallbox sensor entity."""
+
+    min_value: float = 0
+
+
+NUMBER_TYPES: dict[str, NumberEntityDescription] = {
+    CONF_MAX_CHARGING_CURRENT_KEY: WallboxNumberEntityDescription(
+        key=CONF_MAX_CHARGING_CURRENT_KEY,
+        name="Max. Charging Current",
+        device_class=DEVICE_CLASS_CURRENT,
+        min_value=6,
+    )
+}
+
+
 SENSOR_TYPES: dict[str, WallboxSensorEntityDescription] = {
```

Comment:
Could you move `SENSOR_TYPES` to `sensor.py` to address the comment from the last PR: https://github.com/home-assistant/core/pull/58690#discussion_r739772685 ?
Just need to be careful as this will likely create a merge conflict with #58801